build/ci/cloudbuild.push.yaml

```diff
@@ -61,8 +77,11 @@ steps:
         gsutil cp mixer-grpc.$SHORT_SHA.pb gs://datcom-mixer-grpc/mixer-grpc/mixer-grpc.latest.pb
         gsutil acl ch -u AllUsers:R gs://datcom-mixer-grpc/mixer-grpc/mixer-grpc.$SHORT_SHA.pb
         gsutil acl ch -u AllUsers:R gs://datcom-mixer-grpc/mixer-grpc/mixer-grpc.latest.pb
+    waitFor: ["push-image"]
```

Comment:
this one does not need to wait for "push-image"

Reply:
Does it need to wait for "build-proto"?